src/pages/500.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import ErrorPage from "@/promisetracker/components/ErrorPage";
+import backendFn from "@/promisetracker/lib/backend";
+import i18n from "@/promisetracker/lib/i18n";
+import wp from "@/promisetracker/lib/wp";
+
+function GeneralError(props) {
+  return <ErrorPage {...props} />;
+}
+
+export async function getStaticProps({ locale }) {
+  const backend = backendFn();
+  const site = await backend.sites().current;
+  const { navigation } = site;
+  const wpApi = wp();
+  const page = await wpApi.pages({ slug: "error", locale }).first;
+  const posts = await wpApi.pages({ slug: "analysis-articles", locale }).posts;
+  const articles = posts?.slice(0, 4) || null;
```

Comment:
I think we need to check if site has articles enabled and if yes, pull them from the `backend`

Reply:
@gertie-sheshe ?

Reply:
@kilemensi What do you mean by `site has articles enabled`